master/usage/policy/secure-metrics.md

```diff
@@ -0,0 +1,408 @@
+  selector: running-calico == "true"
+  order: 500
+  types:
+  - Ingress
+  ingress:
+  # Deny anything that tries to access the prometheus port
+  # but that doesn't match the necessary selector.
+  - action: Deny
+    protocol: TCP
+    source:
+      notSelector: calico-prometheus-access == "true"
+    destination:
+      ports:
+      - 9093
+```
+
+Then, use calicoctl to apply this policy.
+
+```shell
+calicoctl apply -f typha-prometheus-policy.yaml
+```
+
+## Using a whitelist approach
+
+This approach denies traffic to and from your hosts by default, and requires that all
+desired communication be explicitly allowed by a network policy.
+
+### Overview
+
+The basic process is as follows:
+
+1. Create a default network policy which denies traffic to and from your hosts.
+1. Create host endpoints for each node which you'd like to secure.
+1. Create a network policy which allows desired traffic to the {{site.prodname}} metrics endpoints.
+1. Apply labels to allow access to the prometheus metrics.
+
+### Example for calico/node
+
+1. Create a default network policy to deny host traffic
+
+   We'll start by creating a network policy which will deny all traffic to and from our hosts. This network
+   policy won't have any effect until we create the host endpoints in the next step.
+
+   To do this, create a file named `default-host-policy.yaml` with the following contents.
+
+   ```yaml
+   apiVersion: projectcalico.org/v3
+   kind: GlobalNetworkPolicy
+   metadata:
+     name: default-host
+   spec:
+     # Select all {{site.prodname}} nodes.
+     selector: running-calico == "true"
+     order: 5000
+     ingress:
+     - action: Deny
+     egress:
+     - action: Deny
+   ```
+
+   Then, use calicoctl to apply this policy.
+
+   ```shell
+   calicoctl apply -f default-host-policy.yaml
+   ```
+
+   > **Note**: Calico allows some traffic as a failsafe even after applying this policy. This can
+   > be adjusted using the `failsafeInboundHostPorts` and `failsafeOutboundHostPorts` options
+   > on the [FelixConfiguration resource](../../reference/calicoctl/resources/felixconfig).
+   {: .alert .alert-info}
+
+1. List the nodes on which {{site.prodname}} is running. In this case, we have two nodes in the cluster.
+
+   ```shell
+   $ calicoctl get nodes
+   NAME
+   kubeadm-master
+   kubeadm-node-0
+   ```
+
+1. Create host endpoints for each {{site.prodname}} node.
+
+   Create a file named `host-endpoints.yaml` containing a host endpoint for each node listed
+   above. In this example, the contents would look like this.
+
+   ```yaml
+   apiVersion: projectcalico.org/v3
+   kind: HostEndpoint
+   metadata:
+     name: kubeadm-master.eth0
+     labels:
+       running-calico: "true"
+   spec:
+     interfaceName: eth0
+     node: kubeadm-master
+   ---
+   apiVersion: projectcalico.org/v3
+   kind: HostEndpoint
+   metadata:
+     name: kubeadm-node-0.eth0
+     labels:
+       running-calico: "true"
+   spec:
+     interfaceName: eth0
```

Comment:
Helpful to point out the parts that need to change if your set-up is different?    E.g. "change eth0 to the name of your interface"?